test/rollup.config.sass.js

```diff
@@ -0,0 +1,15 @@
+import scss from '../index.es.js'
```

Comment:
Once this is moved to its own directory, you need to look two directories back. You'll need to do the same for the node-sass test.
```suggestion
import scss from '../../index.es.js'
```
You can also rename this to just `rollup.config.js` after its moved to its own test directory.